examples/pytorch/03-pytorch-gpu-dask-single-model.ipynb

```diff
@@ -4,23 +4,34 @@
+    "![PyTorch and Dask logos](https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/pytorch-dask-logos.png \"doc-image\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
     "\n",
-    "This example generates new pet names by training an LSTM neural network on pet names from Seattle pet license data. The model takes a partially complete name and determines the probability of each possible next character in the name. Then to generate names a character is randomly sampled from that distribution and it's added to the name, then the process is repeated until a stop character is generated.\n",
+    "Training a PyTorch model can potentially be sped up dramatically by having the training computations done on multiple GPUs across multiple workers. This relies on PyTorches [DistributedDataParallel (DDP)](https://pytorch.org/tutorials/intermediate/ddp_tutorial.html) module to take computing the values for each batch and spread them across multiple machines/processors. So each worker computes a part of the batch, and then they are all combined to determine the loss then optimize the nodes. If you kept a network training setup the exact same except tripled the number of GPUs with DDP, you would in practice be using a batch size that is 3x bigger than our original one. Be aware, not all networks benefit from having larger batch sizes, and using PyTorch across multiple workers adds the time it takes to pass the new values between each worker.\n",
     "\n",
-    "## Setting up model training\n",
+    "This example builds on the [introduction to PyTorch with GPU on Saturn Cloud](./01-pytorch-gpu.ipynb) example that trains a neural network to generate pet names. The model uses LSTM layers which are especially good at discovering patterns in sequences like text. The model takes a partially complete name and determines the probability of each possible next character in the name. Characters are randomly sampled from this distribution and added to the partial name until a stop character is generated and full name has been created. For more detail about the network design and use case, see our [Saturn Cloud blog post](https://saturncloud.io/blog/dask-with-gpus/) which uses the same network architecture.\n",
     "\n",
-    "Nothing in this section has anything to do with DDP, Dask, or Saturn. This merely downloads the already cleaned pet names data, creates functions to process it into a format to feed into an LSTM, and defines the model architecture.\n",
+    "_Alternatively, rather than having a Dask cluster be used to train a single PyTorch model very quickly you could have the Dask cluster train many models in parallel. We have a [separate example](./02-pytorch-gpu-dask-multiple-models.ipynb) for that situation.__\n",
```

Comment:
```suggestion
    "_Alternatively, rather than having a Dask cluster be used to train a single PyTorch model very quickly you could have the Dask cluster train many models in parallel. We have a [separate example](./02-pytorch-gpu-dask-multiple-models.ipynb) for that situation._\n",
```